src/frontend/app/shared/components/add-service-instance/select-plan-step/select-plan-step.component.html

```diff
@@ -16,6 +16,11 @@
       <span *ngIf="(getPlanAccessibility(selPlan.entity) | async) !== 'ok'"> ({{ getAccessibilityMessage(selPlan.entity) | async }})</span>
     </app-metadata-item>
     <app-metadata-item label="Free"> {{ isFree(selPlan) | titlecase}} </app-metadata-item>
+    <ng-container *ngIf="!isFree(selPlan)">
```

Comment:
This will always be false as isFree is a string - need to use *ngIf="!selPlan.entity.entity.free">